src/twisted/conch/ssh/keys.py

```diff
@@ -277,13 +277,16 @@ def _fromString_BLOB(cls, blob):
                 curve=b"ecdsa-sha2-nistp256",
             )
             keyObject._sk = True
+            keyObject._application = common.getNS(rest, 3)[2]
```

Comment:
I think that this is now public API..


Assuming that this is implement as a property, this can be

same for the other usage

```suggestion
            keyObject.application = common.getNS(rest, 3)[2]
```